based_build/localization_helper.py

```diff
@@ -0,0 +1,192 @@
+# pyright: reportImplicitOverride=false, reportUninitializedInstanceVariable=false
```

Comment:
the problem is that `textual` allows overrides on some specific methods, which however are not marked `@override`

Reply:
Roughly to say, `textual`'s annotation support is not quite strict enough.

Reply:
the base method doesn't need the `@override` decorator to allow you to override it. by default, everything can be overridden unless decorated with `@final`.

the `@override` decorator instead goes on your method to say that you're intentionally overriding a method on the base class. this helps to prevent issues such as the base class's method being removed/renamed in a future version without you realizing.

i fixed it for you in 9f7419e2b989282653bfa4970e2673f79b33fc43